yaksh/models.py

```diff
@@ -36,10 +36,13 @@
     )
 
 question_types = (
-        ("mcq", "Multiple Choice"),
+        ("mcq", "Single Correct Choice"),
         ("mcc", "Multiple Correct Choices"),
         ("code", "Code"),
         ("upload", "Assignment Upload"),
+        ("integer", "Answer in Integer"),
+        ("string", "Answer in String"),
+        ("float", "Answer in Float"),
```

Comment:
I am not changing this here because this is an options field, the lesser text is better. Having said that, this will be only seen by moderators. Students will see the question type as <b>(FILL IN THE BLANKS WITH INTEGER ANSWER)</b>